lms/services/exceptions.py

```diff
@@ -144,36 +162,23 @@ def _exception_class(response):
         error_description = response_json.get("error_description", "")
 
         if {"message": "Invalid access token."} in errors:
-            return CanvasAPIAccessTokenError
+            return ProxyAPIAccessTokenError
 
         if error_description == "refresh_token not found":
-            return CanvasAPIAccessTokenError
+            return ProxyAPIAccessTokenError
 
         if (
             status_code == 401
             and {"message": "Insufficient scopes on access token."} in errors
         ):
-            return CanvasAPIAccessTokenError
+            return ProxyAPIAccessTokenError
```

Comment:
Might want to move the `CanvasAPIError.raise_from()` class method. Previously it always raised `CanvasAPIError` or a subclass. Now it might raise `ProxyAPIAccessTokenError` which isn't a subclass of `CanvasAPIError`